typegraph/node/sdk/src/runtimes/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+export { WasmEdgeRuntime } from "./wasmedge.js";
```

Comment:
I would say that keeping the previous import style is better.
I am thinking that, for example, we could import potential utility functions `import { SomeRuntime, someUtility1, someUtility2, someType } "@typegraph/sdk/runtimes/some_rt.js"` without having to re-export them.

Reply:
That was because I had some issues for import_map with the `npm:` specifier. I have find a workaround, so I am going to revert that...